packages/components/atoms/f-button/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ v2.0.0
 
 - Return beta to master. Component has JETSansDigital font.
 
+### Removed
```

Comment:
Should this have a minor version bump?

Reply:
This change is already in v2 of f-button, I just added a line to a changelog as forgot to do so in previous PR